content/pt/open-source.md

```diff
@@ -2,16 +2,132 @@
 	"title": "Código aberto",
 	"type": "page",
 	"aliases": ["opensource", "oss", "foss"],
-	"heading": "<span style='text-align: center; font-family: monospace;'>pedidodedados.org <span class='color-red-600' title='amores'>❤</span> código aberto</span>"
+	"heading": "<span style='text-align: center; font-family: monospace;'>pedidodedados.org <span class='color-red-600' title='ama'>❤</span> Código aberto</span>"
 }
 
-<img id="open-source-humaaan" class="top-left-humaaan" src="/img/humaaans/open-source.svg">
+<img id="open-source-humaaan" class="top-left-humaaan" style="margin-top: -70px;" src="/img/humaaans/open-source.svg">
+
+Código aberto está no coração do pedidodedados.org. Nós construímos o projeto do chão com a idéia de ser tão aberto quanto possível. Nossa [constituição]({{< ref "verein/constitution" >}}) nos obriga a publicar todo nosso conteúdo de forma livre e gratuita.
```

Comment:
As we don't currently have a Portuguese translation of the constitution, we have to link to the English version or the build will fail.

```suggestion
Código aberto está no coração do pedidodedados.org. Nós construímos o projeto do chão com a idéia de ser tão aberto quanto possível. Nossa [constituição](https://www.datarequests.org/verein/constitution) nos obriga a publicar todo nosso conteúdo de forma livre e gratuita.
```